main/ejml-dsparse/test/org/ejml/sparse/csc/TestRandomMatrices_DSCC.java

```diff
@@ -62,6 +65,21 @@ void uniform() {
         assertTrue(CommonOps_DSCC.checkSortedFlag(a));
     }
 
+    @ParameterizedTest
+    @ValueSource(strings = {"true", "false"})
+    void generateUniform(boolean sorted) {
+        DMatrixSparseCSC a = RandomMatrices_DSCC.generateUniform(numRows, numRows, 4, -1, 1, sorted, rand);
+
+        assertTrue(CommonOps_DSCC.checkStructure(a));
+    }
+
+    @Test
+    void generateSortedUniform() {
```

Comment:
isn't this covered by the test above?

Reply:
This wasnt supposed to be commited. As the enum.